templates/cicd/manifest.yml

```diff
@@ -0,0 +1,20 @@
+# This manifest defines the relationship and deployment ordering of your environments.
+
```

Comment:
Let's also add a `version` field like we discussed offline to free us from potentially breaking changes.